test/show.jl

```diff
@@ -2325,3 +2325,11 @@ end
     @test_repr "T[1;;; 2 3;;; 4]"
     @test_repr "T[1;;; 2;;;; 3;;; 4]"
 end
+
+@testset "Cmd" begin
+    @test sprint(show, `true`) == "`true`"
+    @test sprint(show, setenv(`true`, "A" => "B")) == """setenv(`true`,["A=B"])"""
+    @test sprint(show, setcpuaffinity(`true`, [1, 2])) == "setcpuaffinity(`true`,[1, 2])"
+    @test sprint(show, setenv(setcpuaffinity(`true`, [1, 2]), "A" => "B")) ==
+          """setenv(setcpuaffinity(`true`,[1, 2]),["A=B"])"""
+end
```

Comment:
I couldn't find where `show(::IO, ::Cmd)` was tested. It looks like the original commit https://github.com/JuliaLang/julia/commit/dee7f6c072eb8389c4858049c26d0d96a94e5f97 didn't add the test for `show` with `setenv`. So, I just threw this in for completeness.